### PR TITLE
wallet and blog pages designed

### PR DESCRIPTION
**Pull request by Falguni-soni**

### CODE REVIEW

File: src/components/addMoney/AddMoney.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        },
+        {
+            amt: '1100'
+        },
+        {
+            amt: '2000'
+        },
+    ]
+
+    const coaches = [
```
Comment (s-prajjwal) [inline src/components/addMoney/AddMoney.jsx:56]: Keep this constant outside of component AddMoney

File: src/components/addMoney/AddMoney.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+import './AddMoney.css'
+
+//bootstrap import
+import "bootstrap/dist/css/bootstrap.min.css";
+import "bootstrap/dist/js/bootstrap.bundle.min";
+import { Footer } from '../footer/Footer'
+
+export const AddMoney = () => {
+
+    const amount = [
```
Comment (s-prajjwal) [inline src/components/addMoney/AddMoney.jsx:17]: Keep this constant outside of component AddMoney

File: src/components/addMoney/AddMoney.jsx (hunk 1 above)
Comment (Falguni-soni) [reply src/components/addMoney/AddMoney.jsx]: Issue resolved.

File: src/components/bestCoaches/BestCoaches.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 15px;
+    text-align: center;
+}
+
+.button {
+    border-radius: 20px;
+    background-color: #EFCC00;
+    border-color: transparent;
+    color: white;
+    font-size: '14px';
```
Comment (s-prajjwal) [inline src/components/bestCoaches/BestCoaches.css:26]: font-size: 14px

File: src/components/coachJourney/CoachJourney.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    margin-left: 30px
+}
+
+.instaIcon {
+    background-color: #F00073;
+    border-radius: 30px;
+    height: 28px;
+    width: 28px;
+    margin-left: 30px
+}
```
Comment (s-prajjwal) [inline src/components/coachJourney/CoachJourney.css:49]: [refactor] common styles can be put in common class name and `background-color` can be put in separate class names.

File: src/components/coachJourney/CoachJourney.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import "bootstrap/dist/css/bootstrap.min.css";
+import "bootstrap/dist/js/bootstrap.bundle.min";
+
+export const CoachJourney = () => {
+    return (
+        <>
+            <NavigationBar />
+            <div className='container-fluid mt-4'>
+                <div className='d-flex justify-content-start align-items-center mx-5'>
+                    <FontAwesomeIcon icon={faGreaterThan} /> &nbsp;&nbsp;
```
Comment (s-prajjwal) [inline src/components/coachJourney/CoachJourney.jsx:21]: Please avoid using `&nbsp`, instead use css for spacings

File: src/components/coachReferral/CoachReferral.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    font-weight: bold;
+}
+
+.tableText {
+    font-size: 18px;
+    color: #000000;
+    font-weight: 500
+}
+
+#btnStyle {
```
Comment (s-prajjwal) [inline src/components/coachReferral/CoachReferral.css:44]: better selector name please

File: src/components/coachVideoChat/CoachVideoChat.css [hunk 7]
```diff
@@ -2,12 +2,13 @@
     background-color: #255B9B;
     border-radius: 10px;
     padding: 4px;
+    font-size: 24
```
Comment (s-prajjwal) [inline src/components/coachVideoChat/CoachVideoChat.css:5]: px?

File: src/components/coachVideoChat/CoachVideoChat.css [hunk 8]
```diff
@@ -2,12 +2,13 @@
...
 }
 
 .card {
     background-color: rgba(129, 186, 255, 0.09) !important;
     height: 550px !important;
-   
+    /* width: 340px; */
```
Comment (s-prajjwal) [inline src/components/coachVideoChat/CoachVideoChat.css:11]: remove commented code

File: src/components/footer/Footer.css [hunk 9]
```diff
@@ -0,0 +1,15 @@
+#footer {
```
Comment (s-prajjwal) [inline src/components/footer/Footer.css:1]: make it class name

File: src/components/footer/Footer.jsx [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+                    <div id='heading'>
+                        Coaches</div>
+                    <div className='mt-3' id='subHeading'> Cricket <br />
+                        Football <br />
+                        Tennis <br />
+                        Yoga <br />
+                        Boxing <br />
+                        Skating <br />
+                        Athletics <br />
+                        BasketBall <br />
```
Comment (s-prajjwal) [inline src/components/footer/Footer.jsx:48]: render it in `divs` with map, avoid using tags like `br`, `hr` etc...

File: src/components/services/Services.jsx [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+                    and track your progress, all in one place.</li>
+                <li>Comprehensive Sport Coverage: Whether you're into mainstream sports or niche disciplines, Athelete's Academy provides
+                    access to a wide range of coaching services.</li>
+            </ul>
+            <div className='pb-5 px-4'
+                style={{
+                    fontSize: '18px',
+                    color: "#000000",
+                    fontWeight: "400",
+                }}>
```
Comment (s-prajjwal) [inline src/components/services/Services.jsx:33]: avoid inline styles everywhere, unless it is just 1 style

File: src/components/walletCoachBalance/WalletCoachBalance.css [hunk 12]
```diff
@@ -0,0 +1,86 @@
+.card {
+    height: 28rem !important;
+    width: 40rem !important;
+    background-color: #FFFFFF !important;
+    border-color: transparent;
+    border-radius: 8px;
+}
+
+#btn {
```
Comment (s-prajjwal) [inline src/components/walletCoachBalance/WalletCoachBalance.css:9]: better id name

File: src/components/walletCoachBalance/WalletCoachBalance.css [hunk 13]
```diff
@@ -0,0 +1,86 @@
+.card {
```
Comment (s-prajjwal) [inline src/components/walletCoachBalance/WalletCoachBalance.css:1]: more specific class name please

File: src/components/walletCoachBalance/WalletCoachBalance.css [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+    border-radius: 10px;
+    font-size: 19px
+}
+
+#aboutText {
+    font-size: 10px;
+    color: #000000;
+}
+
+.secondContainer {
```
Comment (s-prajjwal) [inline src/components/walletCoachBalance/WalletCoachBalance.css:20]: better class name please

File: src/components/walletCoachBalance/WalletCoachBalance.css (hunk 13 above)
Comment (Falguni-soni) [reply src/components/walletCoachBalance/WalletCoachBalance.css:1]: It is a bootstrap class

File: src/components/walletRecharge/WalletRecharge.css [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+    background-image: linear-gradient(#0E86D4, #000000) !important;
+    ;
+}
+
+.vl {
+    border-left: 2px solid #FFFFFF;
+    height: 80px;
+}
+
+.table {
```
[the remark this answers is not in the document]
Comment (Falguni-soni) [reply src/components/walletRecharge/WalletRecharge.css:32]: it is a bootstrap class

File: src/components/blogPage/Blog.css [hunk 16]
```diff
@@ -0,0 +1,23 @@
+.container {
```
[the remark this answers is not in the document]
Comment (s-prajjwal) [reply src/components/blogPage/Blog.css:1]: okay, but why to override bootstrap class? If you want some styles of your own, please make new classes of your own.

File: src/components/walletCoachBalance/WalletCoachBalance.css (hunk 13 above)
Comment (s-prajjwal) [reply src/components/walletCoachBalance/WalletCoachBalance.css:1]: https://github.com/atheltesacademy/WEB-APP/pull/6#discussion_r1531583073

File: src/components/walletRecharge/WalletRecharge.css (hunk 15 above)
[the remark this answers is not in the document]
Comment (s-prajjwal) [reply src/components/walletRecharge/WalletRecharge.css:32]: https://github.com/atheltesacademy/WEB-APP/pull/6#discussion_r1531583073

**Pull request merged**